src/EasyCaching.HybridCache/Configurations/HybridCachingOptions.cs

```diff
@@ -13,6 +13,16 @@ public class HybridCachingOptions
         /// enable logging.
         /// </summary>
         /// <value><c>true</c> if enable logging; otherwise, <c>false</c>.</value>
-        public bool EnableLogging { get; set; } 
+        public bool EnableLogging { get; set; }
+
+        /// <summary>
+        /// local cache provider name
+        /// </summary>
+        public string LocalCacheProviderName { get; set; }
+
+        /// <summary>
+        /// distribute cache provider name
+        /// </summary>
+        public string DistributedCacheProviderName { get; set; }
```

Comment:
Set a defailt value with `EasyCachingConstValue.DefaultRedisName` may be more better.

Reply:
Setting the default provider name does not provide a real service, and it will cause an error without pre-conditions. If setting the default provider name would cause confusion

Reply:
It sounds good!